src/deadline/keyshot_submitter/Submit to AWS Deadline Cloud.py

```diff
@@ -357,7 +357,14 @@ def gui_submit(bundle_directory: str) -> Optional[dict[str, Any]]:
         return None
 
 
-def main():
+def main(lux):
+    if lux.isSceneChanged():
+        lux.getMessageBox(
+            title="Unsaved changes", msg="You have unsaved changes. Save your scene and try again."
```

Comment:
Can we present this more as a "Unsaved changes detected, would you like to save now?" option that calls lux.SaveFile() if the user chooses to save first and then continues submission?